GumDataTypes/GumProjectSave.cs

```diff
@@ -422,21 +422,24 @@ private void PopulateElementSavesFromReferences(string projectRootDirectory, Lin
                 }
             }
 
-            foreach(var reference in BehaviorReferences)
+            if  (BehaviorReferences != null)
```

Comment:
I tried rooting the Gum assemblies from the main project, and setting IsTrimmable on the libraries.
That seems to be XmlSerializer behavior.